criteria/understandable-english-first.md

```diff
@@ -39,7 +39,7 @@ order: 10
 
 ## Management: what you need to do
 
-* Try to limit the use of acronyms, abbreviations, puns or legal/domain specific terms in internal communications in and between teams and stakeholders.
+* Try to limit the use of acronyms, abbreviations, puns or legal/non-English/domain specific terms in internal communications in and between teams and stakeholders. Add anyones that are being used to a glossary and link to it from those places.
```

Comment:
The "anyones" phrasing is clunky. Perhaps it would read better with something more like:

> Create glossary entries for ror these sorts of words and link to the glossary entry from these places they are used.

... still not excellent wording, I know.

Reply:
Perhaps:

> Add any such terms to a glossary and link to it from the places they are being used.

Is that a bit clearer?

Reply:
That is certainly clearer, yes.